### DISCUSSION TITLE
Nothing compiles when using Arduino Uno R4

### DISCUSSION THREAD

**Discussion by Pythagolix** [pschatzmann/arduino-audio-tools#1115] [Q&A] [answered]
I've tried compiling the examples in examples-vs1053 and they all error with error: class 'audio_tools::AnalogConfigStd' does not have any field named 'AnalogConfig'
     AnalogConfigStd(RxTxMode rxtxMode) : AnalogConfig() {

Any ideas what could be the problem?  The plan is to use the Arduino uno r4 minima with music maker shield.  The ada-fruit library also fails to compile., but that sends me down a completely different rabbit hole.  Super frustrating.

**Comment by pschatzmann** [comment, marked as answer]
I could reproduce the issue and have committed a correction
